extensions/amp-script/0.1/amp-script.js

```diff
@@ -133,6 +133,21 @@ export class AmpScript extends AMP.BaseElement {
     });
   }
 
+  /**
+   * @override
+   */
+  onLayoutMeasure() {
+    super.onLayoutMeasure();
+    const {width, height} = this.getLayoutBox();
+    if (width === 0 && height === 0) {
```

Comment:
Note that `onLayoutMeasure` can be called multiple times, including after the amp-script is actually initialized. 

So let's check that `layoutCallback` has not been called yet to avoid emitting this warning incorrectly.

Reply:
You got it

Reply:
Forget to push? :)